Source/IntegrationTests/TestFiles/LitTests/LitTest/comp/placeholders/complex/user.dfy

```diff
@@ -0,0 +1,10 @@
+// RUN: %baredafny run %args %s --target=java --build=%S/Build/build --input %S/wrappers.dfy --input %S/random.dfy --input %S/randomJava.dfy --input %S/Interop/__default.java --spill-target-code > "%t"
+// RUN: %baredafny run %args %s --build=%S/Build/build --input %S/wrappers.dfy --input %S/random.dfy --input %S/randomCSharp.dfy --input %S/Interop.cs --spill-target-code >> "%t"
```

Comment:
You can definitely see how nice it would be to have the project file specify all this, and even better to have a default convention for the code layout of a Dafny project with extern code. :)